.github/workflows/security.yml

```diff
@@ -35,14 +35,14 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout
 
       - name: Run GoSec Security Scanner
-        uses: securego/gosec@master
+        uses: securego/gosec
```

Comment:
my rationale of removing `@master` tags from everywhere is that we can't make sure they keep using `master` branch instead of `main`. My assumption is that not using anything will make it so that it uses the default branch whatever it is.